src/components/RegistrationForm.tsx

```diff
@@ -40,11 +40,11 @@ const RegistrationForm = ({ onValidSubmit } : RegistrationFormProps) : JSX.Eleme
         ev.preventDefault();
     };
 
-    const inputFieldFirstName = <Msg id="components.registrationForm.input_field_first_name"/>;
-    const inputFieldLastName = <Msg id="components.registrationForm.input_field_last_name"/>;
-    const inputFieldEmail = <Msg id="components.registrationForm.input_field_email"/>;
-    const inputFieldPhone = <Msg id="components.registrationForm.input_field_phone"/>;
-    const inputFieldPassword = <Msg id="components.registrationForm.input_field_password"/>;
+    const field1 = <Msg id="forms.reg.fields.firstName"/>;
+    const field2 = <Msg id="forms.reg.fields.lastName"/>;
+    const field3 = <Msg id="forms.reg.fields.email"/>;
+    const field4 = <Msg id="forms.reg.fields.phone"/>;
+    const field5 = <Msg id="forms.reg.fields.password"/>;
```

Comment:
I would try to get rid of these. They are weirdly named and we should avoid passing react elements as props unless that's exactly what we want (in this case it isn't, because we're essentially just sending `<span>`s to be put in a label.)